.github/workflows/test-oblogproxy-ce.yml

```diff
@@ -0,0 +1,126 @@
+name: test oblogproxy-ce
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    paths:
+      - '.github/workflows/**-oblogproxy-ce.yml'
+      - '.github/workflows/**-oceanbase-ce.yml'
+      - 'oblogproxy-ce/**'
+      - 'oceanbase-ce/**'
+      - 'test/**'
+
+concurrency:
+  group: test-oblogproxy-ce-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build-oceanbase-ce:
+    uses: ./.github/workflows/build-oceanbase-ce.yml
+    with:
+      cache_key: test-oblogproxy-ce_oceanbase-ce
+      image_file: oceanbase-ce.tar
+      version: 4.2.1.6-106000012024042515
```

Comment:
the latest version of oceanbase-ce 4.2.1 is 4.2.1.7-107000162024060611

Reply:
The legacy version here is used for cdc libary testing.